exchanges/okcoin/okcoin_websocket.go

```diff
@@ -24,8 +25,53 @@ import (
 	"github.com/thrasher-corp/gocryptotrader/log"
 )
 
+const (
+	// pong message
+	pongBytes = "pong"
+
+	// Public endpoint subscriptions
+	wsInstruments = "instruments"
+	wsTickers     = "tickers"
+	wsCandle3M, wsCandle1M, wsCandle1W, wsCandle1D, wsCandle2D, wsCandle3D, wsCandle5D,
+	wsCandle12H, wsCandle6H, wsCandle4H, wsCandle2H, wsCandle1H, wsCandle30m, wsCandle15m,
+	wsCandle5m, wsCandle3m, wsCandle1m, wsCandle3Mutc, wsCandle1Mutc, wsCandle1Wutc, wsCandle1Dutc,
+	wsCandle2Dutc, wsCandle3Dutc, wsCandle5Dutc, wsCandle12Hutc, wsCandle6Hutc = "candle3M", "candle1M", "candle1W", "candle1D", "candle2D",
+		"candle3D", "candle5D", "candle12H", "candle6H", "candle4H",
+		"candle2H", "candle1H", "candle30m", "candle15m", "candle5m",
+		"candle3m", "candle1m", "candle3Mutc", "candle1Mutc", "candle1Wutc",
+		"candle1Dutc", "candle2Dutc", "candle3Dutc", "candle5Dutc", "candle12Hutc", "candle6Hutc"
+	wsTrades                  = "trades"
+	wsOrderbooks              = "books"
+	wsOrderbooksL5            = "book5"
+	wsOrderbookL1             = "bbo-tbt"
+	wsOrderbookTickByTickL400 = "books-l2-tbt"
+	wsOrderbookTickByTickL50  = "books50-l2-tbt"
+	wsStatus                  = "status"
+	// Private subscriptions
+	wsAccount     = "account"
+	wsOrder       = "orders"
+	wsOrdersAlgo  = "orders-algo"
+	wsAlgoAdvance = "algo-advance"
+)
+
+var defaultSubscriptions = []string{
+	wsTickers,
+	wsCandle1D,
+	wsTrades,
+	wsOrderbooks,
```

Comment:
I think this could be changed to `wsOrderbookTickByTickL400` because its faster

Reply:
this channel wsOrderbookTickByTickL400= "books-l2-tbt" requires authentication. It is not public.

Reply:
> Only API users who are VIP4 and above in trading fee tier are allowed to subscribe to "books-l2-tbt" 400 depth channels 

Ahh yes, I do not have enough money for it 😆 😢 sorry for false-request